server/routes/bookers/booker/bookerDetailsController.test.ts

```diff
@@ -2,17 +2,16 @@ import type { Express } from 'express'
 import request from 'supertest'
 import * as cheerio from 'cheerio'
 import { SessionData } from 'express-session'
-import { appWithAllRoutes, flashProvider } from '../../testutils/appSetup'
+import { appWithAllRoutes, FlashData, flashProvider } from '../../testutils/appSetup'
 import {
   createMockBookerService,
   createMockPrisonerContactsService,
   createMockPrisonService,
 } from '../../../services/testutils/mocks'
 import TestData from '../../testutils/testData'
-import { FlashErrorMessage } from '../../../@types/visits-admin'
 
 let app: Express
-let flashData: Record<string, string | Record<string, string>[] | FlashErrorMessage>
+let flashData: FlashData
```

Comment:
Now define this type in `appSetup.ts` as it was duplicated across all routes test files.